src/ngraph/runtime/gpu/gpu_cuda_kernel_builder.hpp

```diff
@@ -51,7 +51,7 @@ namespace ngraph
 
                 static void get_concat_op(codegen::CodeWriter& writer,
                                           const std::string& name,
-                                          const std::vector<std::string>& data_types,
+                                          const std::string& data_types,
```

Comment:
data_types->data_type for consistency w/ implementation.

Reply:
changed.